src/rgw/motr/synchronization/rgw_motr_gc_sync.h

```diff
@@ -0,0 +1,47 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=2 sw=2 expandtab ft=cpp
+
+/*
+ * Ceph - scalable distributed file system
+ *
+ * SAL implementation for the CORTX Motr backend
```

Comment:
Please refer to `motr/gc/gc.h` file for header comment.

Reply:
done